_posts/2024-04-17-2-dåre-review.md

```diff
@@ -0,0 +1,31 @@
+
+dÅre 2024 gick, som de flesta redan vet, inte till Åre utan till Vemdalen, anledningen till detta? Teorierna är många, såsom att vi skulle tappat bort vägen till Åre eller att vi hamnat i fejd med Åre kommuns IT-avdelning. Men vad det hela handlar om är tyvärr svårt att säga. Hursomhelst, lördag den 13 januari kl. 23.59 från KTH bar det av, De 6 följande timmarna bestod av den tröttaste bussresan en människa kan tänka sig, att det hade varit tentapub, samma lördagkväll hjälpte inte riktigt heller. Under färden gjordes även ett pitstop på macken i den stora metropolen Söderhamn (invånarantal: cirkus lika många som närvarade på MDI-föreläsningarna). Att denna stackars mack kl. 02.34 på morgonen helt plötsligt belägrades av cirkus 100 komiskt sömndruckna dataloger kan nog tillräknas som ett större trauma än de 4 stadsbränderna staden genomgått sedan 1675. Under skrivandet av den här artikeln forskade jag även runt lite om vad för annat som skett i Söderhamn, (där jag bland annat fick reda på att Sveriges första överbefälhavare Olof Thörnell alias Lustmördaren tydligen var från Söderhamn, annan allmän random kuriosa om Lustmördaren är att han 1940 mottog Storkorset av Tyska örnens orden som 1940 gavs ut av Tysklands mest ökända laddtorsk, Adolf Hitler) var det som att någon på P3 börjat få nytta av den nya avlyssningslagen och helt sonika släppte en dokumentär om de s.k. Tingshusmorden som skedde just i Söderhamn. 
+
+<img class="jpg" alt="Sveriges första överbefälhavare Lustmördaren (Olof Thörnell)" src="https://upload.wikimedia.org/wikipedia/commons/1/16/O_G_Thornell.jpg">
+ Sveriges första överbefälhavare Lustmördaren (Olof Thörnell)
+
+Iallafall, resterande bussresa hände det inte så mycket, mest för att klockan var knasigt sent eller tidigt och folk sov eller inbillade sig att de sov för att på så sätt gaslighta sig själva till att inte vara lika trötta. Anyways, väl framme i Vemdalen runt klockan 06.00 gick allt förvånansvärt smidigt. Dock ska det väl nämnas att ena bussen fastnade i snön på parkeringen vilket ledde till att en antal personer behövde hjälpa till att knuffa upp den något som man faktiskt lyckades med utan sådär jättemycket möda och något som busschaufförerna firade genom att sätta fast även den andra bussen i snön. 
+
+<img class="jpg" alt="Dataloger (git) pushing a bus" src="https://yoggi.datasektionen.se/namnder/dbuggen/busspush">
```

Comment:
Omg någon använder yoggi??? Jag behövde logga in för att se den här bilden.
Trots att den här knappen:
![image](https://github.com/datasektionen/dbuggen/assets/32957299/00f94e92-416a-49f9-82f3-79888b1eb455)
ser ut att kopiera en länk direkt till s3 verkar det som att den lömskt nog kopierar en `yoggi.data...`-länk. Om man däremot högerklickar på namnet och trycker på "kopiera länk" e.dyl. i webbläsarens meny så verkar man få en fungerande länk.

Reply:
Gäller btw alla bilder

Reply:
fuuuuuck